src/components/Forms/Radio/style.ts

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components";
+import { BaseRadioProps } from "./index";
+
+type RadioProps = {
+  disabled?: boolean,
+};
```

Comment:
any reason ini di style.ts instead of di type.ts?

perhaps lebih baik dilempar ke type.ts ajaa?

Reply:
ohhh yaa, noted and changed